SimTKcommon/sharedTarget/CMakeLists.txt

```diff
@@ -15,7 +15,11 @@ if(BUILD_UNVERSIONED_LIBRARIES)
     set_target_properties(${SHARED_TARGET} PROPERTIES
         PROJECT_LABEL "Code - ${SHARED_TARGET}"
         SOVERSION ${SIMBODY_SONAME_VERSION})
-
+		
+    if(WIN32 AND ${CMAKE_VERSION} GREATER 3.0) 
```

Comment:
Could you add a comment here and in the staticTarget CMakeLists explaining why this was introduced? Perhaps "Warning 4996 is emitted when using std::copy with raw pointers, as is done in MatrixHelperRep*.h" You could use the longer explanation in the root CMakeLists.txt as well.

Three other changes you could consider: 

1. You could make the version check like [this one](https://github.com/simbody/simbody/blob/master/SimTKcommon/staticTarget/CMakeLists.txt#L25):

```
if(WIN32 AND NOT ${CMAKE_VERSION} VERSION_LESS 3.0.2)
```

2. Also, perhaps say that the purpose of this line is so that downstream targets do not see the warning.

3. Since this warning is emitted only in debug mode, you could use the following so that the flag is only specified when building in debug:

```
target_compile_options(${SHARED_TARGET} INTERFACE
    "$<$<CONFIG:Debug>:/wd4996>")
```

Reply:
Ok, I'll do that.